### ISSUE TITLE
Would it be possible to update the heroquest dice system?

### ISSUE THREAD

**Issue by altair984** [BernhardPosselt/foundryvtt-special-dice-roller#30]
The heroquest dice system is not quite complete, I was wondering if it would be possible to talk to you and explain how to update it. I use it a lot with my friends and we would like to enjoy the system 100% :). Anyway thank you very much for your work.

**Comment by BernhardPosselt** [comment]
Shoot. Ideally you could take a look at it (it's not rocket science) and fill in the remaining parts.

**Comment by altair984** [comment]
The truth is that I have no fucking idea if I'm honest, the most I can do is explain it to you and if you have time and you feel like updating it, things that I would appreciate a lot.
On the other hand if you don't have time I understand and that's ok :D

**Comment by DSBloom** [comment]
What is incomplete about it? I'm putting together HeroQuest in Foundry and was going to use this mod.

**Comment by BernhardPosselt** [comment]
FYI I have no clue and don't run HeroQuest. This feature was proposed by someone else and I implemented it according to their spec.

Maybe the confusion here is about differences between the boardgame and RPG?

**Comment by wgoff194** [comment]
I honestly do not see where it is incomplete. if anything I think I would rather have an option to not get the results from the rolls. can we get an extra /hq tag that gives just the roll faces? or option in the module config? 

I would love to see where this version is falling short

**Comment by BernhardPosselt** [comment]
@wgoff194 what's the use case for not getting the result? Or rather: why do you want that?

**Comment by wgoff194** [comment]
@BernhardPosselt I am using it in a macro where only one side rolls like this: 
![image](https://github.com/BernhardPosselt/foundryvtt-special-dice-roller/assets/29683461/a80f0d25-f643-4281-be7a-c85b98faca68)
and the GM (Zargon) can roll their own defend or attack. I am looking to not have the output under the symbols

**Comment by BernhardPosselt** [comment]
Right, but why do you not want the output under the symbols? Maybe I'm having trouble understanding it because I haven't run the system

**Comment by wgoff194** [comment]
alternatives could be

A) we could have /hq 3ha counts how many skulls0 and 3hd would count how many lion shields. 3ma count monster attacks and 3md count black shields and it only output 1 line underneath with just a count of the requested value. 

B) Keep the data as is but also give counts of shields for each monster(black) or hero(lion) and only show data of either hero or monster if only 1 is requested. i.e. only show hero damage and defence if the command is `/hq 3h`  and hide the monster damage cause it was not called.
